src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/ez-formatted.js

```diff
@@ -0,0 +1,36 @@
+import EzConfigBase from './base';
+
+export default class EzFormattedConfig extends EzConfigBase {
+    constructor(config) {
+        super(config);
+
+        this.name = 'formatted';
+        this.buttons = [
+            'ezmoveup',
+            'ezmovedown',
+            this.getStyles(config.customStyles),
+            // NOTE: alignment currently not supported on <pre> so skipping buttons for that
```

Comment:
@vidarl Actually did not check if this is supported or not, besides noticing there where no handling of this in the xsl. However if we did not have it in legacy either there is probably no need to add it on formatted.

Reply:
We do not have align in legacy ( editor disables align buttons ) on literal.
Docbook does not support it on ProgramListing either, but we are free to add support for `ezxhtml:align` attribute if we choose to.

However, since it was not a need for it in legacy, it is likely no needed now either.